### PR TITLE
Front-end Component Parameters - added validate=options to all fields…

### PR DESCRIPTION
**Pull request by pe7er**
This PR adds validation to parameter fields of type=option to the parameters of .xml files **front-end components**

I've added
```xml
validate="options"
```
to all fields of type=list with defined options

### CODE REVIEW

**Review by zero-24 — APPROVED** [review]
LGTM thanks.

File: components/com_contact/tmpl/category/default.xml [hunk 1]
```diff
@@ -279,6 +300,7 @@
  				type="list"
  				label="COM_CONTACT_NUMBER_CONTACTS_LIST_LABEL"
  				useglobal="true"
+			    validate="options"
```
Comment (Quy) [inline components/com_contact/tmpl/category/default.xml:303]:
```suggestion
 				validate="options"
```

Comment (Quy) [inline components/com_contact/tmpl/category/default.xml]:
```suggestion
				validate="options"
```

Comment (pe7er) [reply components/com_contact/tmpl/category/default.xml]: Thanks @Quy !

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on 443f57a79900fabd5f6b4c4a4718e9db310e1297<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28827">issues.joomla.org/tracker/joomla-cms/28827</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 443f57a79900fabd5f6b4c4a4718e9db310e1297<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28827">issues.joomla.org/tracker/joomla-cms/28827</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28827">issues.joomla.org/tracker/joomla-cms/28827</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**